CHANGELOG.md

```diff
@@ -1,14 +1,9 @@
-### 08/04/2020
```

Comment:
Your fork is outdated.

Reply:
Ok, i try to merge this on my local repo.

Reply:
Should be up to date now.